numba/targets/__init__.py

```diff
@@ -1,3 +1,7 @@
 import sys
 from numba.core.errors import _MovedModule
-sys.modules[__name__] = _MovedModule(locals(), None)
+
+from numba.misc import quicksort
+
+sys.modules[__name__] = _MovedModule(locals(), None,
+                                     extra_alias={'quicksort': quicksort})
```

Comment:
```suggestion
sys.modules[__name__] = _MovedModule(locals(), None)
sys.modules[__name__].quicksort = quicksort
```
then you can skip `extra_alias`.

Reply:
Yes, that's better. I'll change this and remove the changes to `_MovedModule`